src/components/Nav/Nav.js

```diff
@@ -1,7 +1,87 @@
-import React from 'react';
+import { AiOutlineMenu } from 'react-icons/ai';
+import { CgSearch } from 'react-icons/cg';
+import styled from 'styled-components';
+import React, { useState } from 'react';
+import SideDrawer from './SideDrawer';
+import GlobalFonts from '../../fonts/fonts';
 
 const Nav = () => {
-  return <div>Nav</div>;
+  const [navDrawer, setNavDrawer] = useState(false);
+
+  const handleNavDrawer = () => {
+    setNavDrawer(!navDrawer);
+  };
+
+  return (
+    <>
+      <GlobalFonts />
+      <SideDrawer navDrawer={navDrawer} closeDrawer={handleNavDrawer} />
+      <Navigator>
+        <NavWrapper>
+          <NavElement>
+            <NavBtn onClick={handleNavDrawer}>
+              <AiOutlineMenu className="navIcon" />
+            </NavBtn>
+            <NavBtn>
+              <img src="/images/branchTime.png" alt="logo" />
+            </NavBtn>
+          </NavElement>
+          <NavElement>
+            <NavBtn>
+              <NavLogin>시작하기</NavLogin>
+              <CgSearch className="navIcon" />
+            </NavBtn>
+          </NavElement>
+        </NavWrapper>
+      </Navigator>
+    </>
+  );
 };
 
 export default Nav;
+
+const Navigator = styled.section`
+  position: relative;
+  background-color: white 0.5;
+  border-bottom: 1px solid #eee;
+  position: fixed;
+  width: 100vw;
+`;
+
+const NavWrapper = styled.div`
+  ${({ theme }) => theme.flex.flexBox('row', 'center', 'space-between')}
```

Comment:
- 기본 값으로 설정되어 있다면 아래와 같이 하셔도 동일하게 동작합니다.
```suggestion
  ${({ theme }) => theme.flex.flexBox('', '', 'space-between')}
```